src/routes/profileRoutes.ts

```diff
@@ -0,0 +1,13 @@
+// routes/userRoutes.ts
+import express from 'express';
+import { getAllUserProfiles, getUserProfile, updateUserProfile } from '../controllers/profileController';
+
+const router = express.Router();
+router.get('/users/', getAllUserProfiles);
```

Comment:
You will need to remove this since its controller function will also be removed.